pkgs/build-support/dotnet/nuget-to-nix/nuget-to-nix.sh

```diff
@@ -8,6 +8,8 @@ if [ $# -eq 0 ]; then
 fi
 
 pkgs=$1
+tmpfile=$(mktemp /tmp/nuget-to-nix.XXXXXX)
+trap 'rm -f "$tmpfile" EXIT
```

Comment:
It looks like a quote was missed here, breaking the program: `nuget-to-nix: line 25: unexpected EOF while looking for matching ''`
```suggestion
trap 'rm -f "$tmpfile"' EXIT
```